inst/resources/css/default.css

```diff
@@ -69,6 +69,10 @@ pre {
 pre[class] {
   background: #f9f9f9;
 }
+pre.numberSource a.sourceLine {
+  left: 0 !important;
```

Comment:
The use of `!important` is totally justified: I do not know any other mean for controlling this declaration of the **skylighting** stylesheet that is hardcoded [here](https://github.com/jgm/skylighting/blob/a24d0ffe0a0a800642f4da775dadf5825e5740cb/skylighting-core/src/Skylighting/Format/HTML.hs#L154-L155).